source/get-started/virtual-machine-install/virt-manager.rst

```diff
@@ -0,0 +1,234 @@
+   (Intel® VT), on the host system from the UEFI firmware setup.
+
+#. Install the software bundles kvm-host and virt-manager-gui using 
+   :command:`swupd`:
+
+   .. code-block:: bash
+
+      sudo swupd bundle-add kvm-host virt-manager-gui
+
+#. Add your userid to the `kvm` and `libvirt` groups.
+   
+   .. code-block:: bash
+
+      sudo usermod -G kvm -a $USER
+      sudo usermod -G libvirt -a $USER
+
+#. Enable the `libvirtd` daemon and then reboot the system to complete the 
+   process.
+
+   .. code-block:: bash
+
+      sudo systemctl enable libvirtd
+      sudo reboot
+
+   This will add your userid to the specified groups and start the libvirtd 
+   daemon after the system has rebooted.
+
+Download the |CL| installer ISO
+*******************************
+
+There are several options available to set up and use a |CL| VM with
+:command:`virt-manager`. You can either download the `KVM` image and run it as
+is or download the installer ISO and run it to create a new installation of
+|CL|.
+
+This example uses the live-server-installer ISO to create a new installation.
+
+#. Download the `Clear Linux* OS Server` from the `Downloads`_ page. I used
+   Firefox and downloaded it to my `~/Downloads` directory.
+
+#. (Optional) Validate the integrity of the downloaded image by checking the
+   file hash and signatures. Refer to :ref:`validate-signatures` for detailed
+   steps.
+
+Launch and setup virt-manager
+*****************************
+
+Virt-manager is a GUI-based virtual machine manager that runs in your desktop
+environment. I'm using the Gnome desktop and will launch the Virtual Machine
+manager from the applications window. You will see the following application
+window:
+
+.. figure:: figures/virtmgr/virt-manager-01.png
+   :scale: 100%
+   :alt: Virtual Machine Manager
+
+   Figure 1: Virtual Machine Manager
+
+In the `Name` window of the application, select and highlight the `QEMU/KVM`
+item and then select Edit > Connection Details. You will see the dialog box
+showing the `QEMU/KVM Connection Details`.
+
+.. figure:: figures/virtmgr/virt-manager-02.png
+   :scale: 100%
+   :alt: QEMU/KVM Connection Details
+
+   Figure 2: QEMU/KVM Connection Details
+
+On the `Overview` tab, make sure the `Autoconnect` field is checked. Select 
+the
+`Virtual Networks` tab and at the lower left of the dialog window, select the 
+:guilabel:`+` key to add a new network connection. The 
+`Create a new virtual network` dialog window will open as shown in figure 3:
+
+.. figure:: figures/virtmgr/virt-manager-03.png
+   :scale: 100%
+   :alt: Create a new virtual network
+
+   Figure 3: Create a new virtual network
+
+To accept the default values as shown in the figure above, select the 
+:guilabel:`Finish` button to create your virtual network.
+
+Now that you have created your virtual network, you can close the 
+`QEMU/KVM Connection details` dialog box and return to the main screen. You 
```

Comment:
Slight revision for clarity: 
```
... and return to the Virtual Machine Manager main console. 
```
I think it's worth calling this out since "main screen" might confuse readers.  
It's also better as a setup for the next section.